R/derive_joined.R

```diff
@@ -352,6 +352,10 @@ derive_vars_joined <- function(dataset,
 
   if (is.null(new_vars)) {
     new_vars <- chr2vars(colnames(dataset_add))
+    if (any(colnames(dataset_add) %in% colnames(dataset))) {
+      warn_if_vars_exist(dataset, new_vars)
+      warn("Be careful to set appropriate expressions for `by_vars`, `join_vars` and `new_vars`.")
```

Comment:
@bundfussr Do you think this is an appropriate way to address the NULL `new_vars` piece? What're the example use-cases of using this function without `new_vars`

Reply:
A really hacky solution would be to insert like at the end
if (is.null(new_vars)){
  return(
    data %>%
     select(-ends_with(".join")) 
  ) 
}